cmake/OpenCVDetectWebNN.cmake

```diff
@@ -0,0 +1,22 @@
+ocv_clear_vars(HAVE_WEBNN)
+ocv_clear_vars(WEBNN_EMSDK)
+if(WITH_WEBNN)
+  set(WEBNN_HEADER_DIRS "$ENV{WEBNN_NATIVE_DIR}/out/Release/gen/src/include")
+  set(WEBNN_INCLUDE_DIRS "$ENV{WEBNN_NATIVE_DIR}/src/include")
+  set(WEBNN_LIBRARIES "$ENV{WEBNN_NATIVE_DIR}/out/Release/libwebnn_native.so")
```

Comment:
Please use lib name instead of filename, and you should link with "webnn_native" and "webnn_proc".

Reply:
Thanks! But I'm still confused about this. Should I specifically include webnn_proc.h and WebnnNative.h? Besides, the `WEBNN_LIBRARIES` variable in WebGPU's codes is set as `$ENV{WEBGPU_ROOT_DIR}/out/Release/gen/src/dawn` which includes several .cpp, .h, .js files. Is it right? Since I think this variable should link to .so files according to the CMake instruction.